docs/asl/ref/avg.md

```diff
@@ -0,0 +1,102 @@
+Average or mean aggregation operator. There are two variants of the `:avg` operator.
+
+## Aggregation
+
+@@@ atlas-signature
+Query
+-->
+AggregationFunction
+@@@
+
+A helper method that computes the average or mean from one or more time series using the 
+[count](count) aggregate to determine how many time series are present at a time and 
+dividing the sum of the values by the count. This avoids issues where one or time series
+are missing data at a specific time resulting in an artificially low average. E.g. the 
+expression:
```

Comment:
This should maybe include a warning about whether number of time series is a meaningful denominator for the user.

Reply:
Isn't that kinda covered with the `NaN` example below?